package.json

```diff
@@ -11,8 +11,8 @@
     "preview": "vite preview",
     "lint": "eslint --max-warnings 0 \"src/**/*\"",
     "check": "svelte-check --tsconfig ./tsconfig.json && tsc -p tsconfig.node.json",
-    "deploy:dev": "vite build && cd canister && rm -rf src/frontend && mkdir -p src/frontend && cp -r ../dist/* src/frontend/ && dfx deploy",
-    "deploy": "vite build && cd canister && rm -rf src/frontend && mkdir -p src/frontend && cp -r ../dist/* src/frontend/ && dfx deploy --ic"
+    "deploy:dev": "rm -rf node_modules/.vite && vite build && cd canister && rm -rf src/frontend && mkdir -p src/frontend && cp -r ../dist/* src/frontend/ && dfx deploy",
+    "deploy": "rm -rf node_modules/.vite && vite build && cd canister && rm -rf src/frontend && mkdir -p src/frontend && cp -r ../dist/* src/frontend/ && dfx deploy --ic"
```

Comment:
Don't you want to remove the whole node_modules directory? It contains much more cache than just .vite

Reply:
I don't know. This worked to go around the issue I noticed, I propose we go with it. Unless you feel strongly.

Reply:
Actually, I think I don't want to destroy someone's entire `node_module` cache, just wanted to make sure vite will not use any cached artifacts. Initially, I also tried `--force` option for vite but I either used it wrong or it does not work. In any case, I don't plan to spend more time debugging JS.

Reply:
I don't feel strongly, I don't know how these things depend on each other. I only mention the idea because in the thread it seemed like you want to be as conservative as possible with cached things (built artifacts _and_ dependencies), so removing more would be more in line with that. 

But if the current draft already solves the observed problem, great.